CHANGELOG.md

```diff
@@ -9,13 +9,15 @@
 - Command Generate - Improve visibility on type ENUM with one line per value.
 - Readme - Improve the community section.
 - Readme - Remove the Licence section as it it not the right one and it is already accessible in the Github page header.
+- Command Generate - Ask for application host/port when using a connection url.
 
 ### Fixed
 - Command Generate - Fix package name.
 - Command Generate - Remove trailing spaces in `.env`.
 - Technical - Apply ESLint rules to `database-analyzer`.
 - Command Generate - Do not duplicate `createdAt` and `updatedAt` if not named as camelCase in database.
 - Command Generate - Fix generation of field with type array.
+- Command Generate - Fix application host/port usage by asking for it in lumber generate.
 
```

Comment:
This is not mentioned anywhere that this fix install of lumber with custom port and hostname